src/server/fmd/slaughter/slaughterman-name/index.js

```diff
@@ -0,0 +1,48 @@
+import { QuestionPage } from '~/src/server/common/model/page/question-page-model.js'
+import { FmdQuestionPageController } from '~/src/server/fmd/question-page-controller.js'
+import { FullNameAnswer } from '~/src/server/common/model/answer/full-name/full-name.js'
+import { slaughterStubPage } from '../slaughter-stub/index.js'
+
+/** @import { FullNameConfig } from '~/src/server/common/model/answer/full-name/full-name.js' */
+/** @import { ServerRegisterPluginObject } from '@hapi/hapi' */
+
+const questionKey = 'slaughtermanName'
+
+export class Answer extends FullNameAnswer {
+  /** @type { FullNameConfig } */
+  static config = {
+    validation: {
+      firstName: {
+        empty: {
+          message: 'Enter first name'
```

Comment:
I know the ticket has these values but in other tickets we've added more detailed error messages for name/surname, maybe check https://eaflood.atlassian.net/browse/DSFAAP-1577 and ask in the tech channel if we should change them to be consistent?